tests/spark/test_mrjob_spark_harness.py

```diff
@@ -198,5 +215,24 @@ def test_passthru_args(self):
             input_bytes=input_bytes,
             extra_args=['--chars', '--ignore', 'to'])
 
+    def test_combiner_called(self):
+        input_bytes = b'one two three one two three one two three'
+        job = self._harness_job(
+            MRWordFreqCountFailingCombiner, input_bytes=input_bytes)
+
+        with job.make_runner() as runner, \
+                self.assertRaises(Exception) as assert_raises_context:
+            runner.run()
+
+        exception_text = assert_raises_context.exception.__str__()
+        expected_str = MRWordFreqCountFailingCombiner.unique_exception_str
+        assert expected_str in exception_text
+
+    def test_bad_combiner(self):
```

Comment:
Likewise, the contract with the inline runner is just that it imitates Hadoop, which may choose whether or not to run combiners. So comparing output isn't really fair. Instead, just include the expected output in the test case, and explain why you expect this output even though it doesn't match the word frequency count.

It'd probably be a good idea to note if there's another test case in this class that invokes combiners and tests them passively by comparing output.